test/e2e/main_test.go

```diff
@@ -113,7 +113,7 @@ func TestOperatorUpgrade(t *testing.T) {
 	spec := DefaultSpec.DeepCopy()
 	name := strcase.ToKebab(t.Name())
 	spec.ObjectMeta.Name = name
-
+	spec.ObjectMeta.Namespace = tconst.Namespace
```

Comment:
ObjectMeta is redundant
```suggestion
	spec.Namespace = tconst.Namespace
```

Reply:
Or we can define namespace globally for all specs in ObjectMeta struct